framework/src/Volo.Abp.Ddd.Application.Contracts/Volo/Abp/Application/Services/ICreateUpdateAppService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Volo.Abp.Application.Services
+{
+    public interface ICreateUpdateAppService<TEntityDto, in TKey>
+        : ICreateUpdateAppService<TEntityDto, TKey, TEntityDto, TEntityDto>
+    {
+
+    }
+
+    public interface ICreateUpdateAppService<TEntityDto, in TKey, in TCreateUpdateInput>
+        : ICreateUpdateAppService<TEntityDto, TKey, TCreateUpdateInput, TCreateUpdateInput>
+    {
+
+    }
+
+    public interface ICreateUpdateAppService<TGetOutputDto, in TKey, in TCreateUpdateInput, in TUpdateInput>
+        : ICreateAppService<TGetOutputDto, TCreateUpdateInput>,
+            IUpdateAppService<TGetOutputDto, TKey, TUpdateInput>,
+            IApplicationService
```

Comment:
The `ICreateAppService` interface has implemented `IApplicationService`, which can be remove here. 

![image](https://user-images.githubusercontent.com/16813853/84915002-d5915d00-b0ee-11ea-9ab2-7e5253a3ac94.png)